docs/release-checklist.txt

```diff
@@ -16,14 +16,18 @@ Release
 
     git tag -m 'release v1.0' -s -u 925F79DAA74AF221 v1.0
 
-- create source tar archive::
+- create release tar archive::
 
-    git archive --format=tar.gz -o rauc-hawkbit-updater-1.0.tar.gz --prefix=rauc-hawkbit-updater-1.0/ v1.0
+    mkdir build && cd build
+    meson ..
+    ninja dist
```

Comment:
```suggestion
    meson setup build
    ninja -C build dist
```
I think we should be consistent here and turn this into an explicit `meson setup`, let meson create the build dir und let ninja change into it.

Reply:
Agreed.